userChomeHideTabs.css

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* This style will hide the tab bar.
+ * There are two styles, one for MacOS, and one for all other platforms,
```

Comment:
I would like the installation process to be as easy as possible. More specifically, I'm planning on writing a tool to automatically `@import` CSS files in `userChrome.css`, in order to avoid user fiddling in CSS files. Therefore, I would like to ask you to split this into two files.